add_vlan_port.py

```diff
@@ -0,0 +1,28 @@
+#Python   code block   to   add port   to VLAN

```

Comment:
Can we store this file in a sub folder?  Perhaps:
example_code/nas/add_vlan_port.py